libraries/ESP8266WiFi/examples/BearSSL_MaxFragmentLength/BearSSL_MaxFragmentLength.ino

```diff
@@ -0,0 +1,125 @@
+// Shows how to use the Maximum Fragment Length option in
+// BearSSL to reduce SSL memory needs.
+//
+// Mar 2018 by Earle F. Philhower, III
+// Released to the public domain
+
+#include <ESP8266WiFi.h>
+
+const char *ssid = "....";
+const char *pass = "....";
+
+void fetch(BearSSL::WiFiClientSecure *client) {
+  client->write("GET / HTTP/1.0\r\nHost: tls.mbed.org\r\nUser-Agent: ESP8266\r\n\r\n");
+  client->flush();
+  uint32_t to = millis() + 5000;
+  do {
+    char tmp[32];
+    memset(tmp, 0, 32);
+    int rlen = client->read((uint8_t*)tmp, sizeof(tmp) - 1);
```

Comment:
Why is there a -1 here? If it's to assure the null terminator, shouldn't it be assigned explicitly somewhere?

Reply:
`memset` one line back does it, but it's a bad example I agree.  Laziness on my part, combined with C's deficient string handling capabilities.

Will update with a `temp[sizeof(tmp)-1] = 0;` and drop the `memset`.  I think it's repeated in multiple examples as I was losing energy writing them and resorted to cut-n-pastes.  Will check the others, too.

Reply:
No need to change it, I missed the memset :)